README.md

```diff
@@ -2,7 +2,7 @@
 
 The purpose of this effort is to
 develop a standard for [Unicode MessageFormat 2] resources,
-which may be used as a file format or otherwise.
+which may be used as a file format.
```

Comment:
A succinct statement of purpose probably needs to be more general. One issue would be the need to provide a definition for 'resource' early on (not necessarily in this paragraph), since the word has been overloaded elsewhere (notably in the IETF space, e.g. it's the **R** in URI/URL/etc). Here we do not mean "resource == any blob of bytes" but the narrower I18N definition.

---

Perhaps:

```suggestion
Message Resource is an effort to develop
a standard common representation for localizable _resources_,
especially text strings,
which are expected to consist of [Unicode MessageFormat 2] _messages_,
including a file format that incorporates additional information
needed by the localization process or by developers.
```

Reply:
Going to not change this for now, as the next paragraph already defines the meaning of "resource" as it's used here. That may not have been visible in the diff, as it was already in the doc.

Reply:
Having just said I wouldn't change this, I then went ahead and did that, to hopefully clarify the use of message vs. resource.